File: Backend/WebAPI/Requests/CinemaRequest.cs
using System.Collections.Generic;

namespace WebAPI.Requests
{
    public class CinemaRequest
    {
        public string Name { get; set; } = null!;
        public string Description { get; set; } = null!;
        public string CityName { get; set; } = null!;
        public ICollection<ServiceCinemaRequest> ServiceCinemas { get; set; } = null!;
    }
}

File: Frontend/src/app/Requests/CinemaRequest.ts
export class CinemaRequest {
    name : string;
    description : string;
    cityName : string;

    constructor(
        name : string,
        description : string,
        cityName : string) {
        
        this.name = name;
        this.description = description;
        this.cityName = cityName;
    }
}